packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -107,6 +138,25 @@ QUnit.module('MessageBox', moduleConfig, () => {
             assert.strictEqual(onMessageSendStub.callCount, 0);
         });
 
+        QUnit.test('should not be fired on enter key if the textarea input does not contain a value (excluding spaces)', function(assert) {
```

Comment:
Test for testing add line break by enter is missed without call onMessageSend. I mean that we should check input text

Reply:
Please check updated text, I think it's enough now

Reply:
Yes, sure